docs/prometheus-integration.md

```diff
@@ -141,33 +141,51 @@ Below is a list of metrics, provided by the components and by 3rd parties.
 
 #### Antrea Agent Metrics
 
-- **antrea_agent_conntrack_antrea_connection_count:** Number of connections
+- **antrea_agent_conntrack_antrea_connections:** Number of connections
 in the Antrea ZoneID of the conntrack table. This metric gets updated at
 an interval specified by flowPollInterval, a configuration parameter for
 the Agent.
-- **antrea_agent_conntrack_max_connection_count:** Size of the conntrack
+- **antrea_agent_conntrack_max_connections:** Size of the conntrack
 table. This metric gets updated at an interval specified by flowPollInterval,
 a configuration parameter for the Agent.
-- **antrea_agent_conntrack_total_connection_count:** Number of connections
+- **antrea_agent_conntrack_total_connections:** Number of connections
 in the conntrack table. This metric gets updated at an interval specified
 by flowPollInterval, a configuration parameter for the Agent.
-- **antrea_agent_egress_networkpolicy_rule_count:** Number of egress
+- **antrea_agent_egress_networkpolicy_rule_count:** (Deprecated since 1.0.0
+Replace with `antrea_agent_egress_networkpolicy_rules`) Number of egress
 NetworkPolicy rules on local Node which are managed by the Antrea Agent.
-- **antrea_agent_ingress_networkpolicy_rule_count:** Number of ingress
+- **antrea_agent_egress_networkpolicy_rules:** Number of egress
 NetworkPolicy rules on local Node which are managed by the Antrea Agent.
-- **antrea_agent_local_pod_count:** Number of Pods on local Node which are
+- **antrea_agent_ingress_networkpolicy_rule_count:** (Deprecated since 1.0.0
```

Comment:
out of curiosity, why doesn't CI check fail now? Is it because you have marked them as deprecated?

Reply:
I didn't look at the logs, but the CI job (`Kind / Validate metrics in Prometheus document match running deployment's`) is definitely failing right now

Reply:
Sorry I did not realize there is a script to generate the metric doc. Should I update this doc with the correct output of version 1.0.0?

Reply:
You need to make sure your metrics definitions are correct, then run the script and commit the changes

Reply:
I've updated the metrics doc for version v1.0.0 which is generated by the script.  I suspect CI would still fail if the version is not set and defaulting to v0.14.0-dev.

Reply:
That's not how it works. The CI job just makes sure that the script has been run and that the file is up-to-date with the code. There is no dependency on anything else.

Reply:
Sure. I will update the metrics doc as it is.